packages/react-components/src/components/InputBoxButton.tsx

```diff
@@ -41,21 +51,26 @@ export const InputBoxButton = (props: InputBoxButtonProps): JSX.Element => {
   };
   return (
     <TooltipHost hostClassName={inputBoxButtonTooltipStyle} content={tooltipContent} calloutProps={{ ...calloutProps }}>
-      <IconButton
-        className={mergedButtonStyle}
-        ariaLabel={ariaLabel}
-        onClick={onClick}
-        id={id}
-        onMouseEnter={() => {
-          setIsHover(true);
-        }}
-        onMouseLeave={() => {
-          setIsHover(false);
-        }}
-        // VoiceOver fix: Avoid icon from stealing focus when IconButton is double-tapped to send message by wrapping with Stack with pointerEvents style to none
-        onRenderIcon={() => <Stack className={iconWrapperStyle}>{onRenderIcon(isHover)}</Stack>}
-        data-testid={dataTestId}
-      />
+      {/* IconButton doesn't support aria-disabled so adding a parent element that wil have this value set */}
+      {/* in this case aria-disabled will be set to the same value for all children elements */}
+      {/* see aria-disabled documentation for more information */}
+      <Stack aria-disabled={ariaDisabled}>
```

Comment:
Should we use a Stack or just a plain old <div> ?

Reply:
I tried div but noticed that it requires some styles that stack uses as otherwise it takes more space than needed, so used the stack here instead